cmake/projects/benchmark/hunter.cmake

```diff
@@ -8,9 +8,9 @@ include(hunter_pick_scheme)
 
 hunter_add_version(
     PACKAGE_NAME benchmark
-    VERSION "1.2.0"
-    URL "https://github.com/google/benchmark/archive/v1.2.0.tar.gz"
-    SHA1 "5f26619848c59ddc354e2d51e2f196d2a6ddb189")
+    VERSION "1.3.0"
```

Comment:
Add new version instead of overwriting existing one:
* https://docs.hunter.sh/en/latest/creating-new/update.html#update-package